src/modding/documentation/lua.md

```diff
@@ -0,0 +1,959 @@
+This is an automatically generated listing of the new Lua map scripting API, generated for {DEV_VERSION} of OpenRA.
+
+OpenRA allows custom maps and missions to be scripted using Lua 5.1.
+These scripts run in a sandbox that prevents access to unsafe functions (e.g. OS or file access), and limits the memory and CPU usage of the scripts.
+
+You can access this interface by adding the [LuaScript](Traits#luascript) trait to the world actor in your map rules (note, you must replace the spaces in the snippet below with a single tab for each level of indentation):
```

Comment:
This probably won't work?

Reply:
Yes, it fails. But as this is auto-generated, it should be fixed elsewhere I guess.

Reply:
Fixed it manually.